glambie/processing/main.py

```diff
@@ -0,0 +1,71 @@
+import logging
+from glambie.monitoring.logging import setup_logging
+from glambie.config.config_classes import GlambieRunConfig
+from glambie.data.data_catalogue import DataCatalogue
+from glambie.processing.process_region import run_one_region, combine_within_one_region
+from glambie.processing.path_handling import OutputPathHandler, get_output_path_handler_with_timestamped_subfolder
+
+
+log = logging.getLogger(__name__)
+
+
+def run_glambie_assessment(glambie_run_config: GlambieRunConfig,
+                           log_level: str = "INFO",
+                           verbose: bool = True):
+
+    if not log.hasHandlers():
+        setup_logging(log_level=log_level)
+    log.info('Starting GlaMBIE algorithm: log_level=%s, verbose=%s', log_level, verbose)
+
+    if verbose:
+        output_path_handler = get_output_path_handler_with_timestamped_subfolder(glambie_run_config.result_base_path)
+        log.info('Output directory set to %s', output_path_handler.base_path)
+    else:
+        output_path_handler = None
+
```

Comment:
Do we do this here because we intend to run_glambie_assessment() in ways other than just from the command line?

Would be nice to leave the logging setup and the verbose bit to entry_point.main() and then just pass the output_path_handler to this function

Reply:
Sweet!!